packages/webpack-config-terra/src/webpack.config.js

```diff
@@ -227,7 +227,7 @@ const defaultWebpackConfig = (env = {}, argv = {}, options = {}) => {
     resolve: {
       extensions: ['.js', '.jsx'],
       modules: getResolveModules(env),
-      mainFields: ['main'],
+      mainFields: ['browser', 'main'],
```

Comment:
I think these changes make sense but we should confirm this is passive (it's probably not technically passive, but might still be okay), in our repositories. Do you have branches/draft PRs that consume this change to validate it doesn't break the builds? (terra-framework, terra-application, etc...)

Reply:
Yes, this change has been validated in Framework [PR 1353](https://github.com/cerner/terra-framework/pull/1353).